examples/todo-list/src/models/todo-list.model.ts

```diff
@@ -36,3 +36,7 @@ export class TodoList extends Entity {
     super(data);
   }
 }
+
+export interface TodoListRelations {}
```

Comment:
`TodoList` already have some relations defined (I think it has many `TodoListImage` and `Todo` instances), can you add navigational properties for these relations to this interface please?